app/src/test/java/com/rickyhu/hushkeyboard/ui/HushKeyboardUiTest.kt

```diff
@@ -59,6 +59,32 @@ class HushKeyboardUiTest {
         keyButtonShouldBe("R ")
     }
 
+    @Test
+    fun `Second row key should exist, should be enabled, and should have click action`() {
+        keyButtonShouldBe("M".plus(" "))
```

Comment:
Because the `plus()` was originally connected to the notation key list iterator in `forech()`.
I used replace tool in android studio for removing the notation key list, and I forgot to change format.